src/overlays/actors/ovl_En_Honotrap/z_en_honotrap.h

```diff
@@ -6,11 +6,34 @@
 
 struct EnHonotrap;
 
+typedef void (*EnHonotrapActionFunc) (struct EnHonotrap*, GlobalContext*);
+
 typedef struct EnHonotrap {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xF8];
+    /* 0x014C */ EnHonotrapActionFunc actionFunc;
+    /* 0x0150 */ union {
+                    
+                    struct {
+                        ColliderTris tris;
+                        ColliderTrisItem elements[2];
+                    };
+                    ColliderCylinder cyl;
+                } collider;
```

Comment:
maybe we can typedef the union outside the actor struct or something? I really dislike busy actor structs like this that have a ton of stuff going on inside of it 

at the very least tho id remove the line break after the union